src/web/templates/contributing.html

```diff
@@ -0,0 +1,53 @@
+
+{% block page_content %}
+<div class="container">
+  <div class="row g-5">
+    <div class="col-12">
+        <h1 class="text-primary">{{ _("Contributing") }}</h1>
+      <p>
+        {{ _("Subsurface is a community driven open source project. It is available free of charge to everybody, and nobody gets paid to work on it. If you like it and would like to contribute back there are a number of different ways you can do so:") }}
+      </p>
+      <h4>{{ _("By Writing Code") }}</h4>
+      <p>
+        {{ _("If you are a software developer (of any experience level), your help with maintaining and extending Subsurface will be greatly appreciated. Have a look at <a %(link)s>this page</a>.",
+        link="href=https://subsurface-divelog.org/documentation/contributing/") }}
+      </p>
+      <h4>{{ _("By Testing") }}</h4>
+      <p>
+        {{ _("If you would like to work closely with our developers and give them a hand in testing the latest features and bugfixes, please head over to <a %(link)s>this page</a> to find out how to get into contact with us. We especially need people running Windows and Mac (as the majority of the active developers are Linux people).",
+        link="href=https://subsurface-divelog.org/documentation/contributing/") }}
+      </p>
+      <h4>{{ _("By Writing Documentation") }}</h4>
+      <p>
+        {{ _("If you are good with words you can help us by writing, reviewing, and improving documentation and articles for our website. To get started please head over to <a %(link)s>this page</a> to find out how to get into contact with us.",
+        link="href=https://subsurface-divelog.org/documentation/contributing/") }}
+      </p>
+      <h4>{{ _("By Translating Subsurface") }}</h4>
+      <p>
+      {{ _("If you are a non-English-speaker, and would like to be able to use Subsurface in your native language, you can help make this happen by supporting us as a translator. As our translations are centrally handled at <a %(transifex)s>Transifex</a> - please sign up for an account there and then request to join the <a %(transifex_subsurface)s>Subsurface Team</a>.",
+      transifex="href=https://www.transifex.com/",
+      transifex_subsurface="href=https://www.transifex.com/projects/p/subsurface/") }}
+      </p>
+      <h4>{{ _("By Helping Other Users") }}</h4>
+      <p>
+      {{ _("We want to make Subsurface as friendly and welcoming to new users as possible. If you are a seasoned user of Subsurface please join our <a %(link)s>User Forum</a> and provide helpful responses to the questions from new users.",
+        link="href=/user-forum") }}
+      </p>
+      <h4>{{ _("By Supporting Us Financially") }}</h4>
+      <p>
+      {{ _("Running Subsurface costs money. This is particularly the case for the free cloud based synchronisation of dive logs that is built into Subsurface. To cover these expenses we have to rely on donations. If you would like to contribute to cover our cost you can do so <a %(link)s>here</a>.",
```

Comment:
we don't "have to rely on donations" - I have paid for this out of my own pocket for a decade.

Maybe "The principal maintainer of Subsurface and of our infrastructure appreciates donations to help cover these expenses" or something...

Reply:
Fair enough, I didn't know the context. Will update.

Reply:
Thanks for that!